### PR TITLE
HTML of pages with blue banner and very limited styling

### PR DESCRIPTION
**Pull request by DanAWashburn**
This is the basic HTML of four pages. It contains very limited CSS.

### CODE REVIEW

File: checkout.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        <p><a class="subtitle" href="index.html">Home&nbsp;&nbsp;/</a><a class="subtitle" href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a><a class="subtitle" href="blog.html">&nbsp;&nbsp;Blog</a></p>
+      </div>
+    </section>
+    <section id="welcome">
+      <div>
+        <h2>Checkout</h2>
+      </div>
+    </section>
+    <section>
+        <div class="formfield">
```
Comment (ClovisI) [inline checkout.html:36]: In your formfield I would add a couple items.  I would style the labels to have similar coloring as the theme.  In addition, I might change their fonts.

File: checkout.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                    <input type="text" name="city">
+                </label> 
+                <label>State
+                    <select name="state">
+                        <option value=""></option>
+                        <option value="idaho">Idaho</option>
+                        <option value="oregon">Oregon</option>
+                        <option value="washington">Washington</option>
+                    </select>
+                </label>
```
Comment (ClovisI) [inline checkout.html:57]: I'd add a zip code label with input after state.

File: checkout.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  <div class="grid-container">
+    <section id="banner">
+      <div id="letterhead">
+        <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+        <h1>Seattle's premiere dog adoption agency</h1>
+      </div>
+      <div id="cart">
+        <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo"><p id="cartprice">$1234.56</p>
+        <div id="checkout-button">
+          <p>Checkout</p>
```
Comment (ClovisI) [inline checkout.html]: I would suggest putting this between an <a> tag with a link to your checkout.html.  Could apply this change to all of your pages.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  <div class="grid-container">
+    <section id="banner">
+      <div id="letterhead">
+        <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+        <h1>Seattle's premiere dog adoption agency</h1>
+      </div>
+      <div id="cart">
+        <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo">
+        <div id="checkout-button">
+          <p>Checkout</p>
```
Comment (ClovisI) [inline index.html]: I would suggest putting this between an <a> tag with a link to your checkout.html.  Could apply this change to all of your pages.

File: checkout.html (hunk 1 above)
Comment (ClovisI) [reply checkout.html:36]: Another thing you might want try is making a type of container to hold city, state, and zip in the same line as you style this page.

File: index.html (hunk 4 above)
Comment (DanAWashburn) [reply index.html]: Good points, all. Thanks especially for the tips about the form fields, etc. on the checkout page.

**Pull request closed without merging**